# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/emacs-xemacs-shared/no-ce-tags.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- xemacs-21.4.12/lib-src/Makefile.in.in.orig	2003-05-19 23:44:57.000000000 +0200
+++ xemacs-21.4.12/lib-src/Makefile.in.in	2003-05-19 23:52:53.000000000 +0200
@@ -91,7 +91,7 @@
 #ifdef HAVE_MS_WINDOWS
  winclient\
 #endif
- etags ctags b2m ootags
+ b2m ootags
 
 PUBLIC_INSTALLABLE_SCRIPTS=\
 #ifdef INSTALL_GNUSERV
